Nfield.Quota/Builders/QuotaFrameBuilder.cs

```diff
@@ -73,18 +75,23 @@ public QuotaFrameBuilder Target(int? target)
             return this;
         }
 
+        [SuppressMessage("Globalization", "CA1308:Normalize strings to uppercase", Justification = "We want variableName in lowercase and we don't care about locale")]
```

Comment:
I don't think this is right, I think we do care because quota can be in many languages. The way we generate odin variable names here is really not good.

Reply:
We do not care about locale, we just want a consistent comparison that is not case sensitive when you use a character set for odin variable names that the invariant culture can convert to lower case.
The problems that the tolower has (roundtrip related) are not relevant to this use.

Reply:
If the variable names contain spaces (for example) this will not produce the correct result. It is not consistent with what the manager does. I think it is problematic in general, even if the `ToLower` is not. In general we should not suppress warnings IMO.

Reply:
spaces are not allowed and I really hope you are not telling me the manager does a locale aware case insensitive comparison.
And I agree we normally should not ignore warnings, but when the warning is newer then the code and changing the code is risky we should be careful when changing it.

Reply:
The manager generates the odin variable name from the variable name in a more sophisticated way. It does strip away characters that are not valid in an odin variable name.